### ISSUE TITLE
Should we use process time for benchmarks/timings?

### ISSUE THREAD

**Issue by abitofevrything** [ggerganov/whisper.cpp#382]
Currently, the `bench` tool simply spits out the timings from `whisper_print_timings`. These timings are not process specific and as such are influenced by other processes on the system.

Should we use something like `CLOCK_PROCESS_CPUTIME_ID` instead of `CLOCK_MONOTONIC` in `ggml_time_*`? This would make the benchmarks in #89 outdated, but would make future benchmarks more realistic.

**Comment by ggerganov** [comment]
I think we can measure and report both.
On non-Windows system we can use the existing `ggml_cycles()`:

https://github.com/ggerganov/whisper.cpp/blob/d1ea1220ffeff846efd0578e8df3b1d1b802bb5d/ggml.c#L315-L321

**Comment by abitofevrything** [comment]
Yeah, I did see that and also that they aren't used anywhere other than commented out debugging statements - I'll add a field that includes them in `whisper_print_timings` tomorrow.
